main.c

```diff
@@ -423,6 +444,7 @@ void ledSetRow() {
       sdReadTimeout(&SD1, commandBuffer, sizeof(led_t) * NUM_COLUMN + 1, 1000);
```

Comment:
I believe the led_t is now a byte wider and this breaks the interface with the main chip. I don't use this function so might've not noticed that.

Reply:
We might need to do some cleanup and remove this handler -- it is never called from the qmk side. Unless somebody using it in their fork.

Reply:
It's probably there to be able to display notification on the keyboard etc. Not a bad idea, but I guess it lacks some parts. There's hid_listen, and maybe we need hid_speak? That's a bit murky for me.